pulseapi/profiles/serializers.py

```diff
@@ -96,6 +96,9 @@ class UserProfileSerializer(serializers.ModelSerializer):
         allow_blank=True,
     )
 
+    is_active = serializers.BooleanField(
+    )
```

Comment:
this shouldn't be on a new line if it isn't taking any parameters.

Reply:
fixed